SalesforceCommerceCloud/pwa-kit
# [App Extensibility] Generator inserting the default config for each extension (@W-17102676@)

Previously a generated project will have its package.json updated with the names of your selected extensions. This PR now also inserts the default configs for those extensions, so that it's more convenient and visible for the users.

## How to test drive the PR

1. `npm ci` at the root
2. `node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js`
3. During generator's prompt, choose to generate an app (not the app extension) and select to use both existing extensions
4. After the project is generated, look into its package.json
5. Verify that the `mobify.app.extensions` key is filled with 2 extensions and their default configs.

You can also try repeating the above steps, but this time choose to _extract_ the selected extensions.
